### PR TITLE
Release: v0.2.7

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW
 
# Summary
* ADAPT-3897: Accordion component
* ADAPT-3897: Styling for Accordion
* ADAPT-3897: Replacing button tag with SAAButton component
* ADAPT-3897: Fix disappearing - sign
* ADAPT-4388: Add and enable @netlify/plugin-gatsby plugin 
* ADAPT-4430: Add support for github labels to release drafter

# Review By (Date)
- 12:00 Friday the 4th

# Criticality
- Medium

# Review Tasks

## Setup tasks and/or behavior to test

1. Review code
2. Call out any Storyblok changes
3. Review preview build for discrepancies 

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Comment by yvonnetangsu** [comment]
@sherakama I'll add the Accordion component to the PROD SAA space before noon March 4

**Comment by sherakama** [comment]
Thanks and sorry for the short notice. I meant to get this out earlier today.

**Comment by yvonnetangsu** [comment]
> Thanks and sorry for the short notice. I meant to get this out earlier today.

No problem at all! And it's a simple component 😄  Thanks for putting the releases together, @sherakama 🥇

File: src/components/composite/accordion.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  };
+
+  const spacingTopStyle = smallPaddingTop[spacingTop];
+  const spacingBottomStyle = smallPaddingBottom[spacingBottom];
+  const buttonsStyle = 'su-pt-12 su-pb-12 md:su-pt-10 md:su-pb-10 su-rs-ml-1';
+
+  return (
+    <SbEditable content={blok}>
+      <div
+        className={`su-cc
```
Comment (yvonnetangsu) [inline src/components/composite/accordion.js:41]:
```suggestion
        className={`su-w-full
```

Comment (yvonnetangsu) [reply src/components/composite/accordion.js:41]: Fixup for when accordion is placed inside a Grid, and it needs to fit to 100% width even when content is narrower

**Review by yvonnetangsu — APPROVED** [review]
This should be GTG. Accordion and Accordion Item added on PROD SAA Storyblok; also allow for accordion as nested component in Basic Page main content "Content", and as nested block inside Section and Grid "Content" + a preset for a default accordion.

**Comment by sherakama** [comment]
Woot thanks @yvonnetangsu. I'll push this live in the morning.

**Pull request merged**